From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Fri, 26 Jul 2024 15:56:04 -0500
Subject: [PATCH] Solus: Start after systemd-udev

It's weird that this sets WantedBy=multi-user.target and also After=multi-user.target. Don't do that.

---
 res/lactd.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/lactd.service b/res/lactd.service
index 74a2b19..dad8fe9 100644
--- a/res/lactd.service
+++ b/res/lactd.service
@@ -1,6 +1,6 @@
 [Unit]
 Description=GPU Control Daemon
-After=multi-user.target
+After=systemd-udev.service
 
 [Service]
 ExecStart=lact daemon
